src/repart/repart.c

```diff
@@ -7486,6 +7490,19 @@ static int context_fstab(Context *context) {
                 }
         }
 
+        if (arg_merge_fstab) {
+                _cleanup_free_ char *h = NULL;
+
+                r = read_one_line_file(path, &h);
+                if (r > 0 && strcmp(h, fstab_hdr) != 0) {
```

Comment:
please use streq() rather than strcmp() in our codebase

Reply:
Done